enumerables.rb

```diff
@@ -0,0 +1,171 @@
+def my_each
+  return enum_for(__callee__) unless block_given?
+
+  size.times { |i| yield to_a[i] }
+end
+
+def my_each_with_index
+  return enum_for(__callee__) unless block_given?
+
+  length.times { |i| yield(to_a[i], i) }
+end
+
+def my_select
+  return enum_for(__callee__) unless block_given?
+
+  res = []
+  my_each { |i| res << i if yield(i) }
+  res
+end
+
+def my_all
+  return enum_for(__callee__) unless block_given?
+
+  my_each { |i| return false unless yield(i) }
+  true
+end
+
+def my_any
+  return enum_for(__callee__) unless block_given?
+
+  my_each { |i| return true if yield(i) }
+  false
+end
```

Comment:
### Also fix this failing error 
![any](https://user-images.githubusercontent.com/8114528/84962491-ac34f900-b10f-11ea-9ef0-cb24c5b0048e.png)

Reply:
Updated function name